helm/slack-service/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
I'm wondering if we need a separate chart versus just having the slack integration deployed as part of the existing Chart ?

Reply:
We discussed and agreed to keep separate.